src/Perlang.Stdlib/Stdlib/Posix.cs

```diff
@@ -1,62 +1,44 @@
 #pragma warning disable SA1300
+#pragma warning disable SA1629
 #pragma warning disable S3218
 using System;
+using System.ComponentModel;
 using System.Runtime.InteropServices;
 using Perlang.Attributes;
 
 namespace Perlang.Stdlib
 {
     /// <summary>
-    /// Provides support for calling standard POSIX methods.
+    /// Provides support for calling standard POSIX functions.
     ///
     /// The method names are deliberately using "POSIX-style", i.e. short, C-oriented names for methods like `getuid`,
     /// `getpid` and so forth. This is to make it simple for people with experience from POSIX-based systems to find the
     /// method they are looking for.
-    /// </summary>
-    /// <remarks>
-    /// The XML method descriptions are based on [the NetBSD source code](https://github.com/NetBSD/src). The full
-    /// license of these man pages can be found below.
     ///
-    /// There might be subtle differences between systems on some of these methods. For information on how these work on
-    /// e.g. Linux, use a command like `man 2 getgid` (replace `getgid` with the name of the function you are interested
-    /// in).
+    /// This class only contains POSIX-specific functions, i.e. functions available on POSIX-compliant systems like BSD,
+    /// GNU/Linux and macOS, but not available on Windows. For C functions available on all supported platforms, see the
+    /// list below.
     ///
+    /// ### List of methods defined in POSIX but available elsewhere:
     ///
-    /// Copyright (c) 1980, 1991, 1993
-    /// The Regents of the University of California.  All rights reserved.
-    ///
-    /// Redistribution and use in source and binary forms, with or without
-    /// modification, are permitted provided that the following conditions
-    /// are met:
-    /// 1. Redistributions of source code must retain the above copyright
-    ///    notice, this list of conditions and the following disclaimer.
-    /// 2. Redistributions in binary form must reproduce the above copyright
-    ///    notice, this list of conditions and the following disclaimer in the
-    ///    documentation and/or other materials provided with the distribution.
-    /// 3. Neither the name of the University nor the names of its contributors
-    ///    may be used to endorse or promote products derived from this software
-    ///    without specific prior written permission.
+    /// * <see cref="Libc.getcwd"/>
+    /// * <see cref="Libc.getenv"/>
+    /// * <see cref="Libc.getpid"/>
```

Comment:
It may end up being unmanageable to list all `Libc` methods here like this, but... let's give it a try. :+1: